### DISCUSSION TITLE
How to get Country and City names in English (OpenStreetMap)

### DISCUSSION THREAD

**Discussion by SiriusCyberneticsCorporation** [Mapsui/Mapsui#2829] [Q&A] [answered]
Is there a way to force all Country and City names to display an English equivalent in OpenStreetMap?

**Comment by pauldendulk** [comment, marked as answer]
The OpenStreetMap tile layer that we use in most of our samples consists of pre-rasterized tiles and does not have this flexibility. So, you need to search for an alternative tile layer generated like that. There are many alternative tile layers, so I think there is hope you could find one.
